tests/function.py

```diff
@@ -3830,12 +3832,6 @@ def test_upgrade_dependencies_from_venv_EnvBuilder(self):
   def test_issoftkeyword_from_keyword(self):
     self.assertOnlyIn((3, 9), detect("from keyword import issoftkeyword"))
 
-  def test_reversed_from_types_MappingProxyType(self):
-    self.assertOnlyIn((3, 9),
-                      detect("from types import MappingProxyType\n"
-                             "mpt = MappingProxyType({})\n"
-                             "mpt.reversed()"))
```

Comment:
Why was this removed?

https://docs.python.org/3.9/library/types.html?highlight=reversed#types.MappingProxyType

Reply:
Because the fact is that the builtin function `reversed()` now accepts `MappingProxyType` objects as arguments, rather than the `MappingProxyType` having a new `reversed` method.

Reply:
Ah, right. That makes sense.